external/storm-hdfs/src/main/java/org/apache/storm/hdfs/spout/HDFSSpout.java

```diff
@@ -409,9 +409,14 @@ public void open(Map conf, TopologyContext context, SpoutOutputCollector collect
 
     // Reader type config
     if ( readerType==null && conf.containsKey(Configs.READER_TYPE) ) {
-      readerType = conf.get(Configs.READER_TYPE).toString();
+    	String className = (String) conf.get(Configs.READER_TYPE);
```

Comment:
What is the need for this when the spout accepts the readerType via `setReaderType ` ?